app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -92,74 +92,7 @@ p
 
   = render partial: '/shared/legal_terms_for_cc_details'
   = form.actions do
-    = form.commit_button 'Save credit card'
+    = form.commit_button 'Save credit card', button_html: { disabled: true }
 
-/ TODO: All is duplicated. Move this into a pack and use braintree from node_modules
-script src="https://js.braintreegateway.com/web/3.69.0/js/client.min.js"
-script src="https://js.braintreegateway.com/web/3.69.0/js/hosted-fields.min.js"
-javascript:
-  const form = document.querySelector('form#new_customer');
-  const submit = document.querySelector('button[type="submit"]');
-
-  if (typeof braintree !== 'undefined') {
-    braintree.client.create({
-      authorization: '#{j @braintree_authorization}'
-    }, function(clientErr, clientInstance) {
-      if (clientErr) {
-        console.error(clientErr);
-        return;
-      }
-
-      // This example shows Hosted Fields, but you can also use this
-      // client instance to create additional components here, such as:qa
-      // PayPal or Data Collector.
-
-      braintree.hostedFields.create({
-        client: clientInstance,
-        styles: {
-          'input': {
-            'font-size': '14px'
-          },
-          'input.invalid': {
-            'color': 'red'
-          },
-          'input.valid': {
-            'color': 'green'
-          }
-        },
-        fields: {
-          number: {
-            selector: '#customer_credit_card_number',
-            placeholder: '4111 1111 1111 1111'
```

Comment:
does this come from automatically now?

Reply:
Now it comes from https://github.com/3scale/porta/pull/3079/files#diff-e5280eb9a5ca2994755960970425ea7ff092eef504a1a954682533ea1d1cc75c and it's the same as in the Dev portal form.
This was copy-pasted from a Braintree docs example.